pkg/cmd/template/cmd_overlays_test.go

```diff
@@ -146,6 +146,195 @@ yamlfunc: yamlfunc`)
 	}
 }
 
+func TestDocumentOverlaysMapItems(t *testing.T) {
```

Comment:
any reason why do these tests here vs in https://github.com/k14s/ytt/tree/master/pkg/yamltemplate/filetests/ytt-library as individual files?

Reply:
No.  Somehow I missed that whole sub-tree in the project.  I'll move these tests over there.

Hmmm... what factors decide which form each of these tests should take?  (e.g. why even have `cmd_overlays_test.go` in the first place?)

Reply:
cmd_overlays_test.go tests e2e behaviour of overlays as applied to other documents (and some other plumbing surrounding that). no way to do that within single filetest.